content/docs/core-functions/moderation.md

```diff
@@ -3,27 +3,47 @@ title: Moderation
+Most communities self-police in some manner based on seniority. People who’ve been in a community for a long time are often seen as wise and elevated to places of power. Our elders often know a community extremely well. They can help provide context for the various factions within a community, synthesize the diverse opinions those factions represent, and can offer measured responses on the most likely outcomes or best solutions. However these elders *are not infallible* and any community needs to adapt to the changes building in said community from its amorphous organization and reorganization. People join and leave throughout a community's existence. Fresh ideas and new viewpoints can carry a community to new heights and bring important changes. New blood breeds *needed innovation* and *helps to center oppressed or unrepresented voices*. **There needs to be a balance between the old and the new to keep both sides in appropriate check.** If you focus the too much on the opinion of elders, you end up with issues of seniority and cliquey behavior. If you focus too much on the new voices, you risk the community splintering or imploding on itself from a lack of stability or trying to cater to too many conflicting voices at once.
 
 ## On solutions to Power mods
 Solutions to ensure fair moderation often center the voices of the individuals in each community and give them the authority to govern themselves. We want this community to be able to govern itself at multiple levels. However self-governance is difficult. 
 
 ### Elections
-Often people like to point to elections as a way to self-govern. Without even naming some of the issues with elections, we're sure most of you can look at existing elected governmental officials in nearly any country and easily identify at least one person who highlights issues with this process. By its very nature election often becomes a popularity contest, rife for abuse in a plethora of ways that humans with good social skills often use to their advantage.
+Often people like to point to elections as a way to self-govern. Without even naming some of the issues with elections, we're sure most of you can look at existing elected governmental officials in nearly any country and easily identify at least one person who highlights issues with this process. By its very nature an election often becomes a popularity contest, rife for abuse in a plethora of ways that humans with good social skills often use to their advantage.
```

Comment:
By its very nature should have a comma after it

Reply:
Fixed